src/community/json-ld/pom.xml

```diff
@@ -155,6 +155,26 @@
         </dependency>
     </dependencies>
 
+    <build>
+    <plugins>
+    <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.15</version>
+        <configuration>
+            <runOrder>alphabetical</runOrder>
+            <includes>
+                <include>**/*Test.java</include>
+            </includes>
+            <reuseForks>false</reuseForks>
```

Comment:
Yes I can imagine but it's the only way I found right now, to avoid that test method that should use a json-ld template end up using the one from another test.